pkgs/by-name/ne/nextcloud-client-bin/package.nix

```
@@ -0,0 +1,46 @@
+{ p7zip
+, libarchive
+, stdenvNoCC
+, lib
+, fetchurl
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "nextcloud-client-bin";
+  version = "3.13.0";
+  src = fetchurl {
+    url = "https://github.com/nextcloud-releases/desktop/releases/download/v${finalAttrs.version}/Nextcloud-${finalAttrs.version}.pkg";
+    hash = "sha256-xqrOJxkFCQcTN7N0ZEjT85zn+u/o/fJUMZdaMBizls0=";
+  };
+
+  unpackPhase =
+  ''
+    7z x $src
+    bsdtar -xf Payload~
+  '';
+
+  nativeBuildInputs = [
+    p7zip
+    libarchive
+  ];
+
+  installPhase =
+  ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    ls
+    cp -R Applications/Nextcloud.app $out/Applications
+
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://nextcloud.com";
+    description = "Desktop Client for Nextcloud";
+    license = lib.licenses.gpl2Plus;
+    maintainers = [  ];
```

```suggestion
    maintainers = [ ];
```

I would suggest you run `nix run nixpkgs#nixpkgs-fmt -- <path-to-package.nix>` in the future.

---

We don't use nixpkgs-fmt in nixpkgs, fyi

---

> We don't use nixpkgs-fmt in nixpkgs, fyi

Thanks for letting me know, is `nixfmt-rfc-style` then the preferred formatter for nixpkgs?

---

Yes, though it's not enforced treewide as of yet, only in some paths:
https://github.com/NixOS/nixpkgs/blob/75ae7621330ff8db944ce4dff4374e182d5d151f/.github/workflows/check-nix-format.yml#L35-L43